{
    "type": "bundle",
    "id": "bundle--17d6db63-e5af-475b-a810-1789c3bb9934",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--8b768b9d-e6d3-45ed-99b3-e3548a262cf6",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-11-14T00:20:36.655464Z",
            "modified": "2024-11-14T00:20:36.655464Z",
            "name": "CVE-2024-37400",
            "description": "An out of bounds read in Ivanti Connect Secure before version 22.7R2.3 allows a remote unauthenticated attacker to trigger an infinite loop, causing a denial of service.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2024-37400"
                }
            ]
        }
    ]
}